File: VB/Themes/CustomColorScheme.vb
Imports DevExpress.Themes.ColorKeys
Imports DevExpress.Themes.Generic
Imports Windows.UI

Public Class CustomColorScheme
    Inherits GenericColorScheme
    Public Sub New(name As String)
        MyBase.New(name)
    End Sub
    Protected Overrides Sub Initialize()
        MyBase.Initialize()
        AllScopes.RibbonControlColors(RibbonControlColorKeys.HeaderBackground) = New DevExpress.Themes.ColorSet(Color.FromArgb(&HFF, &H41, &H83, &H66), Color.FromArgb(&HFF, &H16, &H4B, &H2F))
        AllScopes.RibbonControlColors(RibbonControlColorKeys.TitleForeground) = New DevExpress.Themes.ColorSet(Colors.Yellow, Colors.Orange)
    End Sub
End Class

File: CS/Themes/CustomColorScheme.cs
using DevExpress.Themes.ColorKeys;
using DevExpress.Themes.Generic;
using Windows.UI;

namespace ColosSchemesDemo1.Themes
{
    public class CustomColorScheme : GenericColorScheme
    {
        public CustomColorScheme(string name) : base(name){}
        protected override void Initialize()
        {
            base.Initialize();
            AllScopes.RibbonControlColors[RibbonControlColorKeys.HeaderBackground]
                = new DevExpress.Themes.ColorSet(Color.FromArgb(0xFF, 0x41, 0x83, 0x66), Color.FromArgb(0xFF, 0x16, 0x4B, 0x2F));
            AllScopes.RibbonControlColors[RibbonControlColorKeys.TitleForeground]
                = new DevExpress.Themes.ColorSet(Colors.Yellow, Colors.Orange);
        }
    }
}
